### ISSUE TITLE
kube-vip manifest creation should have imagePullPolicy configureable

### ISSUE THREAD

**Issue by himslm01** [kube-vip/kube-vip#380]
When creating the demonset manifest for kube-vip by running kube-vip, the resulting manifest currently has the `imagePullPolicy` set to `Always` . I believe that is bad practice, makes the repository a single point of failure, and ignores local images.

I'm using k3s with an air-gap install and I am including kube-vip (v0.4.2) and kube-vip cloud-provider (v0.0.2) images by `docker save`ing the images to a .tar and putting the .tar into `/var/lib/rancher/k3s/agent/images/`.

With `imagePullPolicy` set to `Always` an air-gapped deployment of kube-vip fails to run. I have to pipe the manifest through `sed` to convert `imagePullPolicy` from `Always` to `IfNotPresent`.

I believe it would be useful if a new configuration switch were added to kube-vip's manifest generation to select the appropriate `imagePullPolicy` for this deployment of kube-vip.

This suggestion has been discussed on Slack: https://kubernetes.slack.com/archives/C01RXPHDATB/p1647680137044459

For reference the command lines I currently use to create the kube-vip manifest in the air-gap k3s installation, split into two for understandability, are:

```
sudo ctr --address /run/k3s/containerd/containerd.sock run --rm --net-host \
    ghcr.io/kube-vip/kube-vip:v0.4.2 vip \
    /kube-vip manifest daemonset \
    --arp \
    --interface eth0 \
    --address <FLOATING_IP> \
    --controlplane \
    --leaderElection \
    --taint \
    --inCluster > kube-vip.yaml
```

followed by

```
sed -i "s/imagePullPolicy: Always/imagePullPolicy: IfNotPresent/" kube-vip.yaml
```

**Comment by djpbessems** [comment]
I'm actually surprised that anyone would want the single entry-point to your cluster be dependent on the ability to pull an image every time a node restarts or otherwise reloads; the default `Always` simply doesn't make sense to me.

**Comment by thebsdbox** [comment]
Fixed in v0.8.0
